.github/workflows/earthly-build.yaml

```diff
@@ -48,14 +48,27 @@ jobs:
         run: |
           earthly --ci --output +sources
           earthly --ci +image
-      - name: Push tagged spaceros images to Dockerhub
+      # Only push to latest if we are building a release
+      - name: Set latest image tag
+        id: set_build_tags
+        run: |
+          if [ "${{ github.ref_type }}" == "tag" ]; then
+            echo "latest_image_tag=osrf/space-ros:latest" >> $GITHUB_ENV
+          fi
+        env:
+          latest_image_tag: ""
+      # Push both tagged releases and the latest main builds to Dockerhub
+      - name: Push spaceros images to Dockerhub
+        # TODO: REVERT THIS COMMIT
+        if: ${{ github.ref_type == 'tag' || github.ref_name == '220/merge' }}
```

Comment:
I'm just doing this for now to get a publicly available version for testing the changes in Docker in https://github.com/space-ros/docker/pull/189

Reply:
Image got pushed under the `main` tag: https://hub.docker.com/r/osrf/space-ros/tags

(I didn't want to introduce extra clutter)

Reply:
Tested successfully against the `main` tag in the docker repo's CI: https://github.com/space-ros/docker/actions/runs/11014500722/job/30588908489?pr=189